2018.csv

```diff
@@ -21,14 +21,14 @@ DjangoCon Europe,2018-05-23,2018-05-27,"Heidelberg, Germany",2018-03-01,2018-03-
 PyCon CZ,2018-06-01,2018-06-03,"Prague, Czech Republic",2018-02-28,2018-02-28,https://cz.pycon.org/2018,https://cz.pycon.org/2018/proposals,https://cz.pycon.org/2018/sponsors
 PyLondinium,2018-06-08,2018-06-10,"London, UK",,2018-04-15,https://pylondinium.org/,https://www.papercall.io/pylondinium,https://pylondinium.org/#sponsors
 SciPy,2018-07-09,2018-07-15,"Austin, Texas, USA",2018-02-09,2018-02-09,https://scipy2018.scipy.org,https://scipy2018.scipy.org/ehome/299527/648140/,https://scipy2018.scipy.org/ehome/299527/648129/
-EuroPython,2018-07-23,2018-07-29,"Edinburgh, UK",,,https://ep2018.europython.eu/,,https://ep2018.europython.eu/sponsor
-PyOhio,2018-07-28,2018-07-29,"Columbus, Ohio, USA",,,https://pyohio.org,,
+EuroPython,2018-07-23,2018-07-29,"Edinburgh, UK",,2018-05-20,https://ep2018.europython.eu/,,https://ep2018.europython.eu/sponsor
+PyOhio,2018-07-28,2018-07-29,"Columbus, Ohio, USA",,2018-05-15,https://pyohio.org,,
 Flask Conf Brasil,2018-08-24,2018-08-25,"São Paulo, BR",2018-08-24,2018-06-30,http://flask.python.org.br/,http://flask.python.org.br/,http://flask.python.org.br/
-PyCon Australia,2018-08-24,2018-08-28,"Sydney, NSW, Australia",,,http://2018.pycon-au.org,,
+PyCon Australia,2018-08-24,2018-08-28,"Sydney, NSW, Australia",,2018-04-23,http://2018.pycon-au.org,,
```

Comment:
It looks like this is the CFP start date. From https://2018.pycon-au.org/speak/, the end date is 2018-05-28.

Reply:
:facepalm: